src/libraries/restore/dirs.proj

```diff
@@ -8,6 +8,7 @@
     <Project Include="netstandard/netstandard.depproj" />
     <Project Include="netfx/netfx.depproj" />
     <Project Include="tools/tools.depproj" />
+    <Project Include="runtime/runtime.depproj" />
```

Comment:
this is breaking runtime-live-builds, since artifacts/bin/coreclr is required for this to work.

Reply:
fixed with this: https://github.com/dotnet/runtime/pull/34166/commits/e6d48826b13a0528323840b5bf951b40390baf5a